style.css

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+@media (max-width: 768px) {
```

Comment:
- [ ] We are going to use media queries later in the desktop version, I recommend removing it to avoid issues in the future.